src/components/molecules/Visualizer/Block/HTML/index.tsx

```diff
@@ -105,8 +105,22 @@ const HTMLBlock: React.FC<Props> = ({
 
     // Initialize styles
     frameWindow.document.documentElement.style.margin = "0";
-    frameWindow.document.body.style.color = themeColor ?? getComputedStyle(frameRef).color;
-    frameWindow.document.body.style.margin = "0";
+
+    // Check if a style element has already been appended to the head
+    let style: HTMLElement | null = frameWindow.document.querySelector(
+      'style[data-id="iframe-style"]',
+    );
+    if (!style) {
+      // Create a new style element if it doesn't exist
+      style = frameWindow.document.createElement("style");
+      style.dataset.id = "iframe-style";
```

Comment:
```suggestion
      style.dataset.id = "reearth-iframe-style";
```
I prefer using `reearth` prefix because html may have same name.
And it's useful when we debug it because we can find this attributes is added by our code.